test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -76,7 +76,7 @@ class test_ucp_sockaddr : public ucp_test {
         for (struct ifaddrs *ifa = ifaddrs; ifa != NULL; ifa = ifa->ifa_next) {
             if (ucs_netif_is_active(ifa->ifa_name) &&
                 ucs::is_inet_addr(ifa->ifa_addr)   &&
-                ucs::is_ib_netdev(ifa->ifa_name))
+                ucs::is_rdmacm_netdev(ifa->ifa_name))
```

Comment:
should we be mentioning an rdmacm limitation in a ucp test..?
i mean, when we add another sockaddr transport, this check will exclude the new transport

Reply:
i don't think it's worse than mentioning 'ib', at least it's more accurate
if we add another sockaddr transport will have to update this check

Reply:
IMHO we should be generic, like we do in the sockaddr_accessibility test in test_md.c, but ok, your call.